joomla/joomla-cms
# Try to log errors that hit the global exception handler

#### Summary of Changes

This will try to write an error message about the uncaught Throwable triggering the error page to a logger registered for the `error` category while taking measures to keep the act of logging from triggering a fatal error.
#### Testing Instructions

Set up a logger for the `error` category (the debug plugin should help with this), trigger the error page, and ensure the error gets logged.
